fluent-package/Rakefile

```diff
@@ -411,7 +411,7 @@ class BuildTask
         ENV["GEM_HOME"] = gem_staging_dir
         ENV["INSTALL_GEM_FROM_LOCAL_REPO"] = "yes"
         sh(bundle_command, "_#{BUNDLER_VERSION}_", "install")
-        # Ensure to install binstubs under /opt/td-agent/bin
+        # Ensure to install binstubs under /opt/fluent-package/bin
```

Comment:
In this PR I don't intend to touch the package name and paths in packages, they should be renamed in the next PR.
I wan to ensure keep  builder's code as is as much as possible in this PR.
Is it difficult to keep?

Reply:
okay, I'll fix it.